src/style.css

```diff
@@ -0,0 +1,95 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf) format("opentype");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf) format("opentype");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.search--form--big {
+  height: 70px;
+  margin: 20px 0;
+  background: rgba(255, 255, 255, 0);
+  border: 1px solid #dee3e9;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  position: relative;
+
+}
+
+.search--form--big:hover {
+  background: #ffff;
+  font-style: normal;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search--font:focus {
+  background: linear-gradient(180deg, #ffff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+  font-family: "Avenir", sans-serif;
+  font-weight: bold;
```

Comment:
You should have created something like `Avenir-Heavy`, by this you will avoid specifying `font-weight`